exasol/toolbox/nox/_format.py

```diff
@@ -40,3 +40,9 @@ def fix(session: Session) -> None:
     _version(session, Mode.Fix, PROJECT_CONFIG.version_file)
     _pyupgrade(session, py_files)
     _code_format(session, Mode.Fix, py_files)
+
+
+@nox.session(name="project:format", python=False)
+def fmt_check(session: Session) -> None:
```

Comment:
Please add a function docstring so `nox -l` show's details on what this task is/can be used for.